unime/src/routes/prompt/credential-offer/+page.svelte

```diff
@@ -16,19 +16,17 @@
   import DownloadSimple from '~icons/ph/download-simple-fill';
 
   // TODO: generate binding in core
-  /* eslint-disable @typescript-eslint/no-explicit-any */
-  interface CredentialOffer {
-    credential_issuer: string;
-    credentials: any[];
-    grants: any;
+  interface CredentialConfiguration {
+    display: object;
```

Comment:
I think it should be `display: object[]` since it's a `Vec<serde_json::Value>` in Rust, right?

Reply:
yes you're right, will change it :ok_hand: